<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_form</class>
 <widget class="QWidget" name="Add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить запись</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>выберите...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="photo_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фото</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>26</y>
     <width>240</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Наименование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Размер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Производитель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Поставщик</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Количество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Пол</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="Name_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Size_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Manufacturer_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Provider_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Cost_line"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Num_line"/>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="gender_line"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
